src/Parser/Priest/Shadow/Modules/Core/Haste.js

```diff
@@ -0,0 +1,80 @@
+import CoreHaste from 'Parser/Core/Modules/Haste';
+import SPELLS from 'common/SPELLS';
+
+const debug = false;
+
+const VOIDFORM_HASTE_PER_STACK = 0.01;
+const LINGERING_INSANITY_HASTE_PER_STACK = 0.01;
```

Comment:
Lingering Insanity tooltip just says the Voidform bonus fades over time, so you probably don't need two seperate constants here.

Reply:
Yes, it was only added for clarity reasons. Can be removed if wanted.